addons/project/views/project_sharing_views.xml

```diff
@@ -67,8 +67,8 @@
                                         <field name="milestone_id" />
                                     </span>
                                     <br />
-                                    <t t-if="record.partner_id.value">
-                                        <field name="partner_id"/>
+                                    <t t-if="record.partner_id.value" t-attf-title="#{record.partner_id.value}">
```

Comment:
`t-attf-title` is really needed? 🧐

Reply:
Hey @xavierbol When we truncate the name, the name would appear shorter and hence in order to view the full name on the hover, I have added the `t-attf-title`. So yes it is needed.

Reply:
Hello @mash-odoo ,

Okay, I understood but the `t-attf-title` is added on `t` tag and so I am wondering where that attribute is added since the `t` tag is not a HTML tag. 🤔

Reply:
Hi @xavierbol Oops, my bad. Yes we need to add a span tag for the attribute to be shown.